app/assets/stylesheets/global.scss

```diff
@@ -34,6 +34,7 @@ textarea { padding: 5px; }
 .form-table td input[type='radio'] { width: unset; }
 
 button, .button {
+  // sass-lint:disable-block property-sort-order no-duplicate-properties no-color-literals
```

Comment:
Whyso?

Reply:
I was having an explosion of linter complaints there that I couldn't figure out how to shush.
We might wanna color-strip this actually.